desktop/core/src/desktop/models.py

```diff
@@ -1068,10 +1068,22 @@ def get_home_directory(self, user):
 
   def get_gist_directory(self, user):
     home_dir = self.get_home_directory(user)
-    gist_dir, created = Directory.objects.get_or_create(name=Document2.GIST_DIR, owner=user, parent_directory=home_dir)
-    if created:
-      LOG.info('Successfully created gist directory for user: %s' % user.username)
-    return gist_dir
+    try:
+
```

Comment:
nit: new line above try instead is more logical

Reply:
Fixed